docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -5,110 +5,117 @@ Diffraction Changes
 - :ref:`PolDiffILLReduction <algm-PolDiffILLReduction>` received a number of improvements: changes names of input workspaces to contain polarization information,
   transmission can be provided as a number or a workspace group, new data averaging option depending on measurement 2theta, option to display all measured points
-  on a scatter plot, new option for self-attenuation treatment using measured tranmission.
-- added a 3mf format file describing the PEARL sample and environment shapes for the P-E press. Also fixed a couple of minor issues in the 3mf file format loader used in ref:`LoadSampleEnvironment  <algm-LoadSampleEnvironment>`
-- :ref:`LoadILLDiffraction <algm-LoadILLDiffraction>` now adds input run number also to a metadata field `run_list`, indended to contain a full list of numors, handled by :ref:`MergeRuns <algm-MergeRuns>`
-- :ref:`LoadWANDSCD <algm-LoadWANDSCD-v1>` now has a new option to perform normalization in the same loading process.
+  on a scatter plot, new option for self-attenuation treatment using measured transmission.
+- Several improvements have been made to the group calibration routine including
+
+
+	- More input control parameters, including peak function type for estimating offset after cross correlation and an option to turn on or off the smoothing of data for cross correlation purpose.
+	- The workflow of group calibration script is also polished to make it smoother. Accordingly, unit tests have been updated.
+	- Making it more generic.
+	- Groups are now allowed with dedicated control parameters.
+	- Documentation has been added as a guidance for general users.
+
+- :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` major interface update along with enabling the calibration of T0 and sample position.
+- :ref:`SCDCalibratePanels <algm-SCDCalibratePanels-v2>` minor interface update that allows fine control of bank rotation calibration.
+- :ref:`SNAPReduce <algm-SNAPReduce-v1>` permits saving selected property names and values to file, to aid autoreduction.
+- Add a custom ttmode to the PEARL powder diffraction scripts for running with a custom grouping file.
+- Added a 3mf format file describing the PEARL sample and environment shapes for the P-E press. Also fixed a couple of minor issues in the 3mf file format loader used in :ref:`LoadSampleEnvironment  <algm-LoadSampleEnvironment>`.
 
 Bugfixes
 ########
 - Fix the issue with the calibration diagnostics script when dealing with instruments of which the detector ID does not start from 0.
 - Fix the issue with :ref:`SNSPowderReduction <algm-SNSPowderReduction>` - when invalid height unit is encountered while reading sample log, we should continue by ignoring geometry and rely purely on user input.
-- fix d-spacing calculation when parabolic model is selected.
+- Fix d-spacing calculation when parabolic model is selected.
```

Comment:
This is a bit confusingly worded - I believe it refers to a bug when converting d-spacing to TOF using diffractometer constants with non-zero DIFA - but it might be worth checking the original PR that added this (@DannyHindson might be able to confirm).

Reply:
I've found the PR - #31734. @KedoKudo can you confirm that @RichardWaiteSTFC is correct that this is about a bug when converting d-spacing to TOF using diffractometer constants with non-zero DIFA?

Reply:
Yep so having had a look at the PR (thanks for finding that!) it is indeed to fix conversion from TOF to dSpacing when DIFA non-zero